common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -333,6 +333,16 @@ def tearDown(self):
         super(LoginFailuresAdminTest, self).tearDown()
         LoginFailures.objects.all().delete()
 
+    def test_unicode_username(self):
+        """
+        Test if `__str__` method behaves correctly for unicode username.
+        It shouldn't raise `TypeError`.
+        """
+        try:
+            map(str, LoginFailures.objects.all())
```

Comment:
I'd rather have a more static test like:

`str(LoginFailures(name=whatever))`, that should work much better than `map(...)`.

Reply:
@OmarIthawi, sure thing, I made these casts refer explicitly to `self.user` and `self.user2`.

Btw. we could use the `map` and call `LoginFailures.objects.all().delete()` in `setUpClass` instead to ensure we're relying only on data created in `setUpClass` and `setUp`. What do you think about it?